webapp/src/js/controllers/messages.js

```diff
@@ -84,7 +84,12 @@ angular
           });
         }
       })
-      .catch(err => $log.error('Error fetching contact', err));
+      .catch(function(err) {
+        $log.error('Error fetching contact', err);
+        ctrl.loading = false;
+        ctrl.conversations = [];
+        ctrl.error = 'Error fetching contact';
```

Comment:
The error is rendered by [this code](https://github.com/medic/cht-core/blob/master/webapp/src/templates/directives/messages_list.html#L20) which just checks if it's truthy or not. I think this should just be...

```suggestion
        ctrl.error = true;
```